clients/admin-ui/src/features/common/table/v2/column-settings/ColumnSettingsModal.tsx

```diff
@@ -47,7 +47,8 @@ export const ColumnSettingsModal = <T,>({
         .map((c) => ({
           id: c.id,
           displayText: c.columnDef?.meta?.displayText || c.id,
-          isVisible: c.getIsVisible(),
+          isVisible:
+            tableInstance.getState().columnVisibility[c.id] ?? c.getIsVisible(),
```

Comment:
This is to allow us to set default visibility, see: https://github.com/ethyca/fides/pull/4781/files#diff-cd47c40887e69a55fee3a54b819214c466bf33045c6be67473345c87cef25bbcR983-R986